litr/src/main/java/com/linkedin/android/litr/utils/TranscoderUtils.java

```diff
@@ -109,6 +116,84 @@ public static long getEstimatedTargetVideoFileSize(@NonNull MediaSource mediaSou
         return getEstimatedTargetFileSize(trackTransforms);
     }
 
+    /**
+     * Estimates video track bitrate. On many devices bitrate value is not specified in {@link MediaFormat} for video track.
+     * Since not all data required for accurate estimation is available, this method makes several assumptions:
+     *  - if multiple video tracks are present, average per-pixel bitrate is assumed to be the same for all tracks
+     *  - if both bitrate and duration are not specified for a track, its size is not accounted for
```

Comment:
line 140 if both bitrate and duration; line 144 else, either bitrate or duration is not?

Reply:
Good call, changing the wording